go.mod

```diff
@@ -2,4 +2,7 @@ module gopkg.in/DataDog/dd-trace-go.v1
 
 go 1.12
 
-require github.com/tinylib/msgp v1.1.2
+require (
+	github.com/google/pprof v0.0.0-20210125172800-10e9aeb4a998
```

Comment:
This is probably fine since:
1. This is a hard dependency for the profiler itself.
2. We don't have a pprof integration.

I worry a little bit about requiring an untagged commit (not our fault - `github.com/google/pprof` has no tags) especially for a v0 library where the API may change at any time.

Is this likely to break customer builds? How stable is `github.com/google/pprof`?

In the worst case we could vendor `github.com/google/pprof`.

What do you think?

Reply:
> Is this likely to break customer builds? How stable is github.com/google/pprof?

Hard to tell. That being said, it's probably a bit of an exotic dependency, so I don't think a lot of dd-trace-go users will require it. Therefor I think this is safe enough?

That being said, I've raised in [issue upstream](https://github.com/google/pprof/issues/603) to see if a v1 tag might be possible in the future.

> In the worst case we could vendor github.com/google/pprof.

Yeah. But I guess it also has a bunch of transitive dependencies that we'd have to vendor as well.

```
$ go mod graph | grep google/pprof
gopkg.in/DataDog/dd-trace-go.v1 github.com/google/pprof@v0.0.0-20210125172800-10e9aeb4a998
github.com/google/pprof@v0.0.0-20210125172800-10e9aeb4a998 github.com/chzyer/logex@v1.1.10
github.com/google/pprof@v0.0.0-20210125172800-10e9aeb4a998 github.com/chzyer/readline@v0.0.0-20180603132655-2972be24d48e
github.com/google/pprof@v0.0.0-20210125172800-10e9aeb4a998 github.com/chzyer/test@v0.0.0-20180213035817-a1ea475d72b1
github.com/google/pprof@v0.0.0-20210125172800-10e9aeb4a998 github.com/ianlancetaylor/demangle@v0.0.0-20200824232613-28f6c0f3b639
github.com/google/pprof@v0.0.0-20210125172800-10e9aeb4a998 golang.org/x/sys@v0.0.0-20191204072324-ce4227a45e2e
```

I'll go ahead and squash-merge this now, but if you end up having second thoughts we can revert or fix this before cutting the next release.